docs/.vitepress/Footer.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <footer class="footer">
+    <div class="footer-column logo-column">
+      <img src="/warp-logo.svg" alt="Warp Logo" width="122" />
+    </div>
+    <div class="footer-column">
+      <h3 class="footer-column-title">Collaborate</h3>
+      <ul>
```

Comment:
Our accessability people has in the past adviced against wrapping lists around navigation stuff, I think its mainly because screenreaders read aout so much stuff when it comes to lists. Dont know how relevant it is any more, and I know theres alot of different opinions out there.

Reply:
Ah I didn't know that! 🙈